src/sass/components/_section--promotion.scss

```diff
@@ -0,0 +1,255 @@
+              background: $primary;
+            }
+          }
+        }
+      }
+    }
+
+    .photo {
+      background-size: cover;
+      background-position: center;
+      height: 300px;
+
+      .hover {
+        opacity: 0;
+        height: 100%;
+        transition: .3s;
+
+        &:hover {
+          opacity: 1;
+        }
+
+        .btn-main {
+          margin-top: 100px;
+          text-transform: uppercase;
+          font-weight: 400;
+          border: none;
+        }
+
+        .counter {
+          margin-top: 40px;
+
+          .time {
+            display: inline-block;
+            width: 70px;
+            height: 70px;
+            background: $counter-time-color-background;
+            border-radius: 50%;
+            margin: 2px;
+            color: white;
+            text-transform: uppercase;
+            padding-top: 15px;
+            font-size: 1.5rem;
+            line-height: 1.5rem;
+
+            span {
+              margin-top: 5px;
+              display: block;
+              font-size: 0.9rem;
+              line-height: 8px;
+            }
+          }
+        }
+
+      }
+    }
+
+    .photo1 {
+      background-image: url(../images/promotion/sofa.jpg);
+    }
+
+    .photo2 {
+      background-image: url(../images/promotion/baby.jpg);
+    }
+
+    .photo3 {
+      background-image: url(../images/promotion/room.jpg);
+    }
+
+    .bottom-bar {
+
+      p {
+        font-weight: 700;
+        font-size: 1.2em;
+        margin: 10px 0;
+        text-align: center;
+      }
+
+      .stars {
+        text-align: center;
+
+        a {
+          text-decoration: none;
+          color: $text-color;
+          display: inline-block;
+          font-style: normal;
+          font-variant: normal;
+          text-rendering: auto;
+          line-height: 1;
+          font-family: 'Font Awesome 5 Free';
+          font-weight: 400;
+
+          &.full {
+            font-weight: 900;
+          }
+          &:before {
+            content: "\f005";
+          }
+        }
+      }
+
+      .slider {
+        width: 100%;
+        height: 1px;
+        background: black;
+        margin: 15px 0;
+        position: relative;
+
+        .slide1 {
+          width: 20%;
+          height: 5px;
+          background: $primary;
+          position: absolute;
+          top: 0;
+          left: 40%;
+        }
+      }
+      .price {
+        display: flex;
+        align-items: center;
+        position: absolute;
+        right: 15px;
+        bottom: 0;
+
+        .price-old {
+          font-size: 13px;
+          color: $promotion-dots-color;
+          padding: 5px 10px;
+          text-decoration: line-through $text-color;
+        }
+      }
+    }
+
+    .furniture {
+      max-height: 490px;
+      height: 490px;
+      padding: 0;
+
+      .carousel-inner {
+        position: relative;
+        width: 100%;
+        height: 100%;
+        overflow: hidden;
+      }
+      .furniture1 {
```

Comment:
Just giving an alternative: you could use here `:nth-of-type` pseudoselector. It is easier to extend, for example when you want to add another row of furniture, you just need to duplicate current row (with `furniture` classes) and in CSS you need to add three `.furniture:nth-of-type(X)` selectors, where `X` is nth elem.

BTW IMHO it's better to have images as `img`s than backgrounds because these images are website's content, not a decoration.